apps/native-component-list/webpack.config.js

```diff
@@ -274,6 +321,10 @@ module.exports = {
     useWebModule('Performance/Systrace', 'Performance/Systrace'),
     useWebModule('HMRLoadingView', 'Utilities/HMRLoadingView'),
     useWebModule('RCTNetworking', 'Network/RCTNetworking'),
+
+    new WorkboxPlugin.GenerateSW(),
```

Comment:
@fson @ide any thoughts on this service worker? It spits out a lot of comments (probably configurable) but then it also cuts down on a lot of SW complexity.

Reply:
It's a Google thing as well btw

Reply:
I think it’s OK to have this in a template or maybe baby’s first PWA and it’s also important to make it easy to remove/replace.